app/index.html

```diff
@@ -87,13 +85,13 @@
 							</select>
 						</div>
 					</div>
-					<div id='command-text-{{$index}}'>
-						{{empireId}} {{action}} {{boardSquare.territory.territory_id}} {{destination}} {{source}}
-					</div>
+					<blockquote ng-model='command' id='command-text-{{$index}}'>
+						{{action}} {{boardSquare.territory.territory_id}} {{destination}} {{source}}
+					</blockquote>
 				</form>
 			</li>
 		</ul>
-		<input class="btn btn-primary" type="submit">
+		<input class="btn btn-primary" type="submit" ng-click="matchesCtrl.sendOrders(matchId, empireId, 'move 2 1')">
```

Comment:
You shouldn't need to ever specify the controller like this.  You're function sendOrders is a function on `$scope`, no?  As such, as long as your controller is applied to this page, it should be within scope.